src/components/search-results/index.js

```diff
@@ -188,8 +188,10 @@ class SearchResults extends Component {
     } = this.props;
     const searchItems = items.filter(item => !item.related);
     const relatedItems = items.filter(item => item.related && item.type !== 'filter');
-    const message = searchItems.length > 0 ? 'You might also be interested in...' : `Looks like we don't have any results that match your search. But you might be interested in...`;
-
+    // we might want to have this depend on the browser language at some point:
+    // const message = searchItems.length > 0 ? 'You might also be interested in...' : `Looks like we don't have any results that match your search. But you might be interested in...`;
+    const message = searchItems.length > 0 ? 'Måske er du også interesseret i...' : `Din søgning gav ingen resultater, men måske er du interesseret i ...`;
+    // see: https://github.com/numo-labs/isearch-ui/issues/257
```

Comment:
Do the ellipsis make sense in danish?

Also instead of the full stops: `...` we should use the utf-8 character `…`